notebooks/tutorial_calendar_shifter.ipynb

```diff
@@ -0,0 +1,1266 @@
+{
```

Comment:
<p>You can pass an 'ax' object to visualize as well. This way you can plot multiple calendars in one figure, and have them nicely stacked on top of each other with a shared axis.</p><pre class="ql-syntax" spellcheck="false">import matplotlib.pyplot as plt
import numpy as np
fig, axes = plt.subplots(nrows=4, sharex=True)
for i, cal in enumerate(cal_list):
&nbsp; &nbsp; cal.visualize(n_years=1, relative_dates=False, show_length=True, ax=axes[i])
axes[-1].set_xlim([np.datetime64("2021-05-12"), np.datetime64("2021-08-01")])
fig.autofmt_xdate()
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI4S2S/s2spy/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>